src/main/java/com/sprints/services/SprintsService.java

```diff
@@ -19,8 +18,8 @@
 	public SprintDomain updateSprint(SprintDomain sprintDomain, String id);
 
 	public List<SprintDomain> findAllByParams(Optional<String> name, Optional<String> technology,
-			Optional<LocalDate> start_date, Optional<LocalDate> end_date);
+			Optional<String> start_date, Optional<String> end_date);
 
 	public List<SprintDomain> findAllSprints(Optional<String> name, Optional<String> technology,
-			Optional<LocalDate> start_date, Optional<LocalDate> end_date);
```

Comment:
LocalDate to String ?

Reply:
I had to change this and handle dates different because on Metrics API they handle dates a string and so entering an invalid date format will no longer cause exception, but instead just ignore the parameter